docs/articles/advanced_topics/generic_host.md

```diff
@@ -0,0 +1,142 @@
+        .RunConsoleAsync();
+}
+```
+This code does a few things. First, it calls the `Host.CreateDefaultBuilder()` method, which creates a new `IHostBuilder` instance with some default settings. Then, it calls the `UseConsoleLifetime()` method to configure the lifetime of the host. This tells the host to keep running until it receives a [SIGINT or SIGTERM Signal](https://en.wikipedia.org/wiki/Signal_(IPC)#SIGINT) i.e. when the user presses Ctrl+C in the console, or when another program tells it to stop.
+
+Next, it configures the services that the host will use by calling the `ConfigureServices()` method. In this case, it adds a new `BotService` service, which is a class that you'll need to define next.
+
+Finally, it calls the `RunConsoleAsync()` method to start the host and begin running your service. That's it! With just a few lines of code, you can use the .NET Generic Host to run your application as a service.
+
+## Setting up your Service
+You'll need to create a class which implements the `IHostedService` interface, which defines two methods: `StartAsync()` and `StopAsync()`. These methods are called by the host when your application starts and stops respectively.
+
+```cs
+public sealed class BotService : IHostedService
+{
+    private readonly ILogger<BotService> _logger;
+    private readonly IHostApplicationLifetime _applicationLifetime;
+    private readonly DiscordClient _discordClient;
+
+    public BotService(ILogger<BotService> logger, IHostApplicationLifetime applicationLifetime)
+    {
+        this._logger = logger;
+        this._applicationLifetime = applicationLifetime;
+        this._discordClient = new(new()
+        {
+            Token = "YourBotTokenHere",
+            TokenType = TokenType.Bot,
+            Intents = DiscordIntents.AllUnprivileged
+        });
+    }
+
+    public async Task StartAsync(CancellationToken token)
+    {
+        await _discordClient.ConnectAsync();
+        // Other startup things here
+    }
+
+    public async Task StopAsync(CancellationToken token)
+    {
+        await _discordClient.DisconnectAsync();
+        // More cleanup possibly here
+    }
+}
+```
+>[!WARNING]
+> Hard-coding your bot token into your source code is not a good idea, especially if you plan to distribute your code publicly. This is because anyone with access to your code can easily extract your bot token, which can be used to perform unauthorized actions on your bot account.
+>Instead, it's recommended that you store your bot token in a secure location, such as a configuration file, environment variable, or secret storage service. You can then retrieve the token at runtime and pass it to the initializer. See for example [How to use a configuration with GEneric Host](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-7.0) or [how to use an environment variable](https://learn.microsoft.com/en-us/dotnet/api/system.environment.getenvironmentvariable?view=net-7.0). 
```

Comment:
"How to use a configuration with generic host", aside from having a capitalization typo, sounds a little awkward. Maybe "How to consume configuration with the Generic Host"?